README.md

```diff
@@ -76,7 +76,7 @@ while (!hosebirdClient.isDone()) {
 You can close a connection with
 
 ```java
-hosebirdClient.shutdown();
+hosebirdClient.close();
```

Comment:
I don't see `close()` on `Client` or `BasicClient`. Should this be `hosebirdClient.stop()` instead?